ext/opcache/zend_accelerator_module.c

```diff
@@ -609,7 +609,7 @@ static ZEND_FUNCTION(opcache_get_status)
 	array_init(return_value);
 
 	/* Trivia */
-	add_assoc_bool(return_value, "opcache_enabled", ZCG(enabled) && (ZCG(counted) || ZCSG(accelerator_enabled)));
+	add_assoc_bool(return_value, "opcache_enabled", ZCG(enabled) && ZCG(accelerator_enabled));
```

Comment:
Can the `ZCG(enabled) &&` part be dropped?

Reply:
yes. right.